librz/bin/format/spc700/spc_specs.h

```diff
@@ -1,3 +1,4 @@
+// SPDX-FileCopyrightText: maijin <maijin@malware.lu>
```

Comment:
@Maijin do you want to use another email here?

Reply:
Do I need to use an email?

Reply:
If not just remove it